assets/cluster-version-operator/cluster-version-operator-deployment.yaml

```diff
@@ -7,16 +7,29 @@ spec:
   selector:
     matchLabels:
       k8s-app: cluster-version-operator
+      clusterID: {{ .ClusterID }}
```

Comment:
You cannot change the label selector for a deployment.  This is a general comment.  Here is an example failure: `The Deployment "restricted" is invalid: spec.selector: Invalid value: v1.LabelSelector{MatchLabels:map[string]string{"foo":"bar", "run":"restricted"}, MatchExpressions:[]v1.LabelSelectorRequirement(nil)}: field is immutable`

Reply:
Updated to remove label selector